developer_README.md

```diff
@@ -94,118 +59,15 @@ Nothing else.
-
-`./appspec.json`: define the settings UI on MoveApps. Users of your App can enter their configuration values.
-
-![img.png](documentation/app-configuration-ui.png)
-
-You can also use our [Settings Editor](https://www.moveapps.org/apps/settingseditor) to generate the App configuration
-
-```
-"settings": [
- {
-   "id": "line_width",
-   "name": "Line width",
-   "description": "The width of the lines in the plot.",
-   "defaultValue": 2,
-   "type": "INTEGER"
- },
- {
-   "id": "legend",
-   "name": "Include legend?",
-   "description": "Should the plot contain a legend?",
-   "defaultValue": false,
-   "type": "CHECKBOX"
- }
-],
-```
-
-`./app-configuration.json`: this is only needed during the app development to simulate an App run
-
-```
-{
-  "line_width": 2,
-  "legend": true
-}
-```
-
-`./RFunction.R`: your App will be called with the user's App configuration
-
-```
-rFunction = function(data, line_width, legend, ...) {
-}
-```
-
-`./tests/testthat/test_RFunction.R`: do not forget to test your App
-
-```
-test_data <- test_data("input3_stork.rds")
-
-test_that("happy path", {
-  actual <- rFunction(data = test_data, sdk = "unit test", year = 2005)
-  expect_equal(unique(lubridate::year(actual@timestamps)), 2005)
-})
-
-test_that("year not included", {
-  actual <- rFunction(data = test_data, sdk = "unit test", year = 2023)
-  expect_null(actual)
-})
-```
-
-### Produce an App artefact
-
-Your App can write files which the user can download after it has run.
-
-`./appspec.json`
-
-```
-  "createsArtifacts": true,
-```
-
-`./RFunction.R`
-
-```
-pdf(appArtifactPath("MorningReport_overviewTable.pdf"), paper = "a4r")
-```
-
-**Notice:** Only files are permitted to act as MoveApps App artifact! If your app produces a directory as an App artificat you have to bundle it eg. by zipping it. In other words: at the moment your App completes its work there must be only files present in `APP_ARTIFACTS_DIR`.
-
-#### example for zipping:
-```
-library('zip')
-dir.create(targetDirFiles <- tempdir())
-...
-# add any files to targetDirFiles
-...
-zip_file <- appArtifactPath(paste0("myfiles.zip"))
-zip::zip(zip_file, 
-    files = list.files(targetDirFiles, full.names = TRUE),
-    mode = "cherry-pick")
-```
-
-## Include files to your App
-
-[Details and examples about _auxiliary files_](https://docs.moveapps.org/#/auxiliary).
-
-This template also implements in `./RFunction.R` a showcase about this topics.
-
----
 
 ## R packages management / renv (optional)
 
 The template is prepared to use [`renv` as a dependency manager](https://rstudio.github.io/renv/articles/renv.html) - but is disabled by default (_opt-in_).
 You can [activate `renv` with `renv::activate()`](https://rstudio.github.io/renv/articles/renv.html#uninstalling-renv) and then use it in the [usual `renv` workflow](https://rstudio.github.io/renv/articles/renv.html#workflow).
 
+
 ### Docker support (optional)
 
 - at the end your app will be executed on MoveApps in a Docker container.
```

Comment:
```suggestion
- Your app will be executed on MoveApps in a Docker container. Specially for debugging errors that are not straight forward, it can be very useful to execute your App in a docker environment. These more complex error are often due to system libraries that need to be installed in MoveApps to run the App. The easiest way to find out which ones they are is to run the App in a docker environment.
```